### ISSUE TITLE
Biopython incompatibility

### ISSUE THREAD

**Issue by shenwei356** [fastqe/fastqe#30]
:(

BioPython 1.78

```
$ fastqe 
Traceback (most recent call last):
  File "/bin/fastqe", line 5, in <module>
    from fastqe.fastqe import main
  File "/usr/lib/python3.9/site-packages/fastqe/fastqe.py", line 25, in <module>
    from Bio.Alphabet import IUPAC
  File "/usr/lib/python3.9/site-packages/Bio/Alphabet/__init__.py", line 20, in <module>
    raise ImportError(
ImportError: Bio.Alphabet has been removed from Biopython. In many cases, the alphabet can simply be ignored and removed from scripts. In a few cases, you may need to specify the ``molecule_type`` as an annotation on a SeqRecord for your script to work correctly. Please see https://biopython.org/wiki/Alphabet for more information
```

**Comment by lonsbio** [comment]
Should be fixed now in `v0.2.7` [release](https://github.com/fastqe/fastqe/releases/tag/v0.2.7), also on PyPI.

**Comment by shenwei356** [comment]
Bioconda is not synced :(
